Gemfile

```diff
@@ -26,4 +26,18 @@ if RUBY_VERSION < '2.0'
   gem 'mime-types', '<3.0', :require => false
 end
 
+group :system_tests do
+  if beaker_version = ENV['BEAKER_VERSION']
+    gem 'beaker', *location_for(beaker_version)
+  end
+  if beaker_rspec_version = ENV['BEAKER_RSPEC_VERSION']
+    gem 'beaker-rspec', *location_for(beaker_rspec_version)
```

Comment:
You have duplicate entry for "beaker-rspec". Only specify the dependency one time.